tests/e2e/improved_csi_idempotency.go

```diff
@@ -304,6 +304,23 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-file-vanilla] "+
 		extendVolumeWithServiceDown(serviceName, namespace, client, storagePolicyName, scParameters,
 			volumeOpsScale, true, isServiceStopped, c)
 	})
+
+	/*
+		Create volume when SPS goes down
+		1. Create a SC using a thick provisioned policy
+		2. Create a PVCs using SC
+		3. Bring down sps service and wait for 5mins (default provisioner timeout)
+		4. Bring up storage-quota-webhook
+		5. Wait for pvcs to be bound
+		6. Delete pvcs and SC
+		7. Verify no orphan volumes are left
+	*/
+	ginkgo.It("create volume when storage-quota-weebhook goes down", func() {
```

Comment:
This testcase will run for vanilla as well, we can add a tag to just run for supervisor suite

Reply:
Also please correct the test steps accordingly

Reply:
done